File: CS/FilterDuplicateRecords_Unbound/MainViewModel.cs
using DevExpress.Mvvm;
using System.Collections.ObjectModel;

namespace FilterDuplicateRecords_Unbound
{
    public class MainViewModel : ViewModelBase
    {
        public ObservableCollection<Item> Source { get; } = new ObservableCollection<Item>(Item.GetData(1000));
    }
}

File: VB/FilterDuplicateRecords_Unbound/MainViewModel.vb
Imports DevExpress.Mvvm
Imports System.Collections.ObjectModel

Namespace FilterDuplicateRecords_Unbound

    Public Class MainViewModel
        Inherits ViewModelBase

        Public ReadOnly Property Source As ObservableCollection(Of Item) = New ObservableCollection(Of Item)(Item.GetData(1000))
    End Class
End Namespace
